CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## Bolt Next
+
+### New features
+
+* **Support for multiple configuration files** ([#608](https://github.com/puppetlabs/bolt/issues/608))
```

Comment:
```suggestion
* **Support for user-level and system-wide configuration files** ([#608](https://github.com/puppetlabs/bolt/issues/608))
```

Reply:
This is nit-picky, but we don't support merging arbitrary configuration files - we still load config from specific places, now there's just more than one config file that gets loaded :P

Reply:
Not not-picky at all! :) Updated.